src/_designs/components/generation/index.vue

```diff
@@ -8,11 +8,11 @@
             fill="black"
           />
         </div>
-        <BlockHeight :value="data.key_block.height" />
+        <BlockHeight :value="data.keyBlock.height" />
       </div>
       <div class="container-first-inner">
         <Account
-          :value="data.key_block.beneficiary"
+          :value="data.keyBlock.beneficiary"
           title="Sender"
```

Comment:
The title will be `Beneficiary`, not `Sender`

Reply:
updated